bundle.Dockerfile

```diff
@@ -28,17 +27,17 @@ LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1 \
     operators.operatorframework.io.bundle.channels.v1=alpha \
     operators.operatorframework.io.bundle.channel.default.v1=alpha
 
-# This label gets replaced by render_templates, so it is safer to keep it on its own
-LABEL version="v0.0.0"
-
-# This label maps to the brew build target
-LABEL com.redhat.component="windows-machine-config-operator-bundle-container"
-
-# This last block are standard Red Hat container labels
+# This block are standard Red Hat container labels
 LABEL \
     name="openshift4/windows-machine-config-operator-bundle" \
```

Comment:
Can we combine this? i.e.`LABEL name="openshift4/windows-machine-config-operator-bundle" \`

Reply:
I guess we can we had taken this from a document Confluence document that specifies required labels.

Reply:
updated.

Reply:
Yes, I had copied this from the confluence doc and I did not want to change things around at that point. Now that we are more confident around the build no reason to blindly follow their formatting unless it breaks something.